docs/en/Running-Alluxio-on-Virtual-Box.md

```diff
@@ -34,19 +34,28 @@ Install [python>=2.7](https://www.python.org/), not python3.
 
 Under `deploy/vagrant` directory in your home directory, run:
 
-{% include Running-Alluxio-on-Virtual-Box/install-vagrant.md %}
+```
+bash
```

Comment:
this should be 
<pre>```bash
</pre>

Reply:
bash-orders changed in docs/en/Running-Alluxio-on-Virtual-Box.md